baselines/dom.generated.d.ts

```diff
@@ -200,11 +200,12 @@ interface ComputedEffectTiming extends EffectTiming {
     progress?: number | null;
 }
 
-interface ComputedKeyframe extends Record<keyof CSSStyleDeclaration, string> {
-    composite?: CompositeOperation | null;
-    computedOffset?: number;
-    easing?: string;
-    offset?: number | null;
+interface ComputedKeyframe {
+    composite: CompositeOperationOrAuto;
+    computedOffset: number;
+    easing: string;
+    offset: number | null;
+    [property: string]: string | number | null | undefined;
```

Comment:
>I have no idea how to represent this sort of open dictionary with a few typed keys

How about using `extends Partial<Record<keyof CSSStyleDeclaration, string | number | null | undefined>>`, while it's somewhat long?

Reply:
Does that work for `{ '--variable': '5px' }` though?

Reply:
Ah, right, custom properties. 😭